src/AutoAdmin.tsx

```diff
@@ -238,7 +238,7 @@ const attributeToInput = (input: AutoAdminAttribute) => {
       return <BooleanInput label={input.label} source={input.attribute} options={input.fieldOptions} />;
     case Date:
       return input.fieldOptions && input.fieldOptions.showTime ? (
```

Comment:
```suggestion
      return input.fieldOptions && input.fieldOptions.showTime ? (
```
`input.fieldOptions.showTime` or `! input.fieldOptions.showTime`